blueprints/app/files/package.json

```diff
@@ -12,17 +12,17 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint": "concurrently --group --raw \"npm:lint:*(!fix)\"",
```

Comment:
- `--group`: orders the output _as if_ the commands are run sequentially
- `--raw`: only log the raw output of tasks (no prefixes)

This improves readability of the output.

Reply:
The error case:
![image](https://github.com/ember-cli/ember-cli/assets/199018/62b6f2e7-0b47-4c52-bff9-429e1a76ce0d)

The nothing is wrong case is pretty nice: 
![image](https://github.com/ember-cli/ember-cli/assets/199018/b33b4bec-5923-4b0d-b49e-8993ef5797b7)


but I think the error case is more confusing?